sample_project/settings.py

```diff
@@ -84,7 +84,10 @@
         'USER': os.getenv('DB_USER', 'mentira'),
         'PASSWORD': os.getenv('DB_PASS', 'mentira'),
         'HOST': os.getenv('DB_HOST', 'localhost'),
-        'PORT': os.getenv('DB_PORT', '5432')
+        'PORT': os.getenv('DB_PORT', '5432'),
+        "OPTIONS": {
+            "application_name": os.getenv("APPLICATION_NAME", "mentirinha"),
```

Comment:
Since this is a database config and all database env vars are namespaced to `DB_`, I would use `DB_APPLICATION_NAME` as the env var name.

Reply:
Great review, it doesn't make sense not to have the DB_ prefix which is a database configuration